sklearn/linear_model/_base.py

```diff
@@ -46,6 +46,31 @@
 # intercept oscillation.
 
 
+def _deprecate_normalize(normalize, default):
```

Comment:
I'm not sure we need this ATM? i understand this might be useful once we deal with different default values but we should probably introduce this when we need it

Reply:
Thanks @NicolasHug 
There are 21 more linear model which need this deprecation but we didn't want to make one huge PR which would be more difficult to review. This PR will show an example to other PRs:
#17746, #17750, #17769, #17772, #17785
they are all going to use  _deprecate_normalize(normalize, default) defined here.